YAML/tests/syntax_test_flow.yaml

```diff
@@ -112,47 +113,53 @@ continuation"
+#   ^ meta.mapping.value string.unquoted.plain.in -entity.name.tag
+#    ^ meta.mapping.yaml punctuation.separator.mapping
+#      ^^ meta.mapping.key.yaml string.unquoted.plain.in
+#        ^ meta.mapping.yaml punctuation.separator.key-value.mapping
+#          ^^ meta.mapping.value.yaml string.unquoted.plain.in
 #            ^ punctuation.separator.mapping
-# <- meta.flow-mapping punctuation.definition.mapping.begin
-*alias, e : f }
-#^^^^^          variable.other.alias
-#     ^         punctuation.separator.mapping
-#       ^       string.unquoted.plain.in entity.name.tag
-#        ^      -string
-#         ^     punctuation.separator.key-value.mapping
-#           ^   string.unquoted.plain.in, -entity.name.tag
-#             ^ meta.flow-mapping punctuation.definition.mapping.end
+# <- meta.mapping.yaml punctuation.definition.mapping.begin
+*alias, true : false }
+#^^^^^^^^^^^^^^^^^^^^^ meta.mapping - meta.mapping meta.mapping
+#^^^^^ variable.other.alias
+#     ^ punctuation.separator.mapping
+#       ^^^^ meta.mapping.key.yaml constant.language.boolean.yaml - string
+#            ^     punctuation.separator.key-value.mapping
+#              ^^^^^ meta.mapping.value.yaml constant.language.boolean.yaml
+#                    ^ meta.mapping.yaml punctuation.definition.mapping.end
+
+{ 1: 2 }
 
 {? !!str 'key' : !!str value,
-#^^^^^^^^^^^^^^^^^^^^^^^^^^^  meta.flow-pair.explicit
-#               ^^^^^^^^^^^^  meta.flow-pair.value
-#                           ^ punctuation.separator.mapping -meta.flow-pair
-#^                            punctuation.definition.key-value.begin
-#  ^^^^^                      storage
-#        ^^^^^                string
-#        ^                    punctuation
-#            ^                punctuation
-#              ^              punctuation.separator.key-value.mapping
-#                ^^^^^        storage
-#                      ^^^^^  string
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.mapping - meta.mapping meta.mapping
+#               ^^^^^^^^^^^^ meta.mapping.value
+#                           ^ punctuation.separator.mapping - meta.mapping.value
+#^ punctuation.definition.key.begin.yaml
+#^^^^^^^^^^^^^^ meta.mapping.key.yaml
+#  ^^^^^ storage
+#        ^^^^^ string
+#        ^ punctuation
+#            ^ punctuation
+#              ^ meta.mapping.yaml punctuation.separator.key-value.mapping
+#                ^^^^^ storage
+#                      ^^^^^ string
  ? !!str foo : :bar
-#^ punctuation.definition.key-value.begin
+#^^^^^^^^^^^^ meta.mapping.key.yaml
+#^ punctuation.definition.key.begin.yaml
 #  ^^^^^ storage
-#        ^^^ string entity.name.tag
-#            ^ punctuation.separator.key-value.mapping -string
-#              ^ string.unquoted.plain.in
+#        ^^^ string.unquoted.plain.in.yaml
+#            ^ meta.mapping.yaml punctuation.separator.key-value.mapping.yaml
+#              ^^^^ meta.mapping.value.yaml string.unquoted.plain.in.yaml
 }
 
+# TODO catch `!!set` and set meta.set scope
```

Comment:
Were you going to leave this TODO here, or fix it in another PR?

Reply:
Definitely not in this PR.